onnxruntime/test/providers/internal_testing/internal_testing_tests.cc

```diff
@@ -350,6 +350,5 @@ TEST(InternalTestingEP, TestOrtModelWithCompileFailure) {
     ExecuteMnist(session, true /* enable_custom_ep */);
   }
 }
-
```

Comment:
nit: stray change ?

Reply:
There was an accidental include of a temporary test in a previous iteration. Must have removed an empty line when I undid that.